### PR TITLE
Update CPack packaging for ROCm release support

### PR DESCRIPTION
**Pull request by dgaliffiAMD**
Look for ROCM_LIBPATH_VERSION environment variable to patch the CPACK_PACKAGE_VERSION.
Add some status logs to output packaging information.

### CODE REVIEW

File: cmake/ConfigCPack.cmake [hunk 1]
```diff
@@ -200,24 +198,45 @@ set(CPACK_RPM_PACKAGE_RELEASE
...
+if(DEFINED ENV{CPACK_DEBIAN_PACKAGE_RELEASE})
+    set(CPACK_DEBIAN_PACKAGE_RELEASE $ENV{CPACK_DEBIAN_PACKAGE_RELEASE})
+endif()
+
+if(DEFINED ENV{CPACK_RPM_PACKAGE_RELEASE})
+    set(CPACK_RPM_PACKAGE_RELEASE $ENV{CPACK_RPM_PACKAGE_RELEASE})
+endif()
+
+message(STATUS "Packaging Information ... ")
+message(STATUS "  CPack package name: ${CPACK_PACKAGE_NAME}")
```
Comment (jrmadsen) [inline cmake/ConfigCPack.cmake:230]: I believe all of these should use the `omnitrace_add_feature(<VARIABLE> “<DESCRIPTION>")` function

**Review by jrmadsen — CHANGES_REQUESTED** [review]

File: cmake/ConfigCPack.cmake (hunk 1 above)
Comment (dgaliffiAMD) [reply cmake/ConfigCPack.cmake]: Using the output summary from the Omnitrace features, rather than the messages.

**Review by jrmadsen — APPROVED** [review]
We should probably enable [CPACK_DEBIAN_PACKAGE_SHLIBDEPS](https://cmake.org/cmake/help/latest/cpack_gen/deb.html#variable:CPACK_DEBIAN_PACKAGE_SHLIBDEPS) and [CPACK_RPM_PACKAGE_AUTOREQ](https://cmake.org/cmake/help/latest/cpack_gen/rpm.html#variable:CPACK_RPM_PACKAGE_AUTOREQ) to auto-generate the dependencies (+ the similar cpack variables for generating what the package provides) but that should be in another PR.

**Comment by dgaliffiAMD** [comment]
Thanks for the feedback. I can make these changes in a future update.

**Pull request merged**